apps/database/supabase/migrations/20260421120000_add_member_type_to_workspace_invite_links.sql

```
@@ -0,0 +1,43 @@
+  wil.id,
+  wil.ws_id,
+  wil.code,
+  wil.creator_id,
+  wil.max_uses,
+  wil.expires_at,
+  wil.created_at,
+  wil.updated_at,
+  wil.type,
+  COUNT(wilu.id) AS current_uses,
+  CASE
+    WHEN wil.expires_at IS NOT NULL AND wil.expires_at < now() THEN true
```

SQLFluff (4.1.0)

[error] Unnecessary CASE statement. Use COALESCE function instead.

(ST02)